.circleci/config.yml

```diff
@@ -117,6 +115,7 @@ commands:
                 name: Install site
                 command: |
                   ddev composer config repositories.dkan path dkan
+                  ddev composer require --dev osteel/openapi-httpfoundation-testing -W
```

Comment:
This should be a dependency in getdkan/recommended-project and/or dkan core.

Reply:
done